scss/components/_buttons.scss

```diff
@@ -285,14 +286,20 @@
 // Styleguide components.buttons.icon-buttons
 
 .button--search {
-  @include u-background-image('i-search--primary');
+  @include u-icon-button($magnifying-glass);
+  background-size: 2.2rem;
```

Comment:
The icons themselves are exported at the same size, but in different instances we want the icons to be bigger or smaller than their standard size.